style.css

```diff
@@ -0,0 +1,139 @@
+#top-nav {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 50px;
+}
+
+#top-nav a {
+  text-decoration: none;
+  font-family: Poppins, sans-serif;
```

Comment:
- [ ] [OPTIONAL] It's great if you import Poppins fonts from google to either your css or html file for users that don't have poppins installed on their system.

## EXAMPLE

`@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;1,100;1,200;1,300;1,500;1,600;1,700&display=swap");
`